src/Sarif/CodeGenHints.json

```diff
@@ -339,10 +339,9 @@
         "flags": true,
         "zeroValueName": "None",
         "memberNames": [
-          "SuppressedInSource",
-          "SuppressedInBaseline"
+          "SuppressedInSource"
```

Comment:
we are so proud of ourselves. it's almost as good an idea as Tuple<T1>

---
In reply to: [63094872](https://github.com/Microsoft/sarif-sdk/pull/264#discussion_r63094872) [](ancestors = 63094872)